### PR TITLE
Reformat code radioList. A clearer idea.

### PR DESCRIPTION
**Pull request by githubjeka**

### CODE REVIEW

File: framework/helpers/BaseHtml.php [hunk 1]
```diff
@@ -974,27 +976,32 @@ public static function radioList($name, $selection = null, $items = [], $options
...
-        } else {
-            $tag = isset($options['tag']) ? $options['tag'] : 'div';
+        if (!isset($options['tag'])) {
+            return $hidden . static::tag('div', implode($separator, $lines), $options);
         }
 
-        unset($options['tag'], $options['unselect'], $options['encode'], $options['separator'], $options['item'], $options['itemOptions']);
-
-        if ($tag === false) {
+        if ($options['tag'] == false) {
```
Comment (SilverFire) [inline framework/helpers/BaseHtml.php]: `===`

File: framework/helpers/BaseHtml.php [hunk 2]
```diff
@@ -929,8 +929,8 @@ public static function checkboxList($name, $selection = null, $items = [], $opti
      * @param array $options options (name => config) for the radio button list container tag.
      * The following options are specially handled:
      *
-     * - tag: string|boolean, the tag name of the container element. Disable the container of the
-     *   radio buttons if it is false.
+     * - tag: string|false, string is the tag name of the container element. Disable the container of the radio buttons
```
Comment (SilverFire) [inline framework/helpers/BaseHtml.php]: I'm suggesting another description: `False to render radio buttons without container`

File: framework/helpers/BaseHtml.php [hunk 3]
```diff
@@ -974,27 +976,32 @@ public static function radioList($name, $selection = null, $items = [], $options
...
-        if ($tag === false) {
+        if ($options['tag'] == false) {
             return $hidden . implode($separator, $lines);
-        } else {
-            return $hidden . static::tag($tag, implode($separator, $lines), $options);
         }
+
+        $tag = $options['tag'];
+        unset($options['tag']);
+        return $hidden . static::tag($tag, implode($separator, $lines), $options);
```
Comment (SilverFire) [inline framework/helpers/BaseHtml.php]: I think we can store `implode($separator, $lines)` in some variable not to duplicate it 3 times

**Comment by SilverFire** [comment]
I like it more, thank you @githubjeka

File: framework/helpers/BaseHtml.php (hunk 1 above)
Comment (githubjeka) [inline framework/helpers/BaseHtml.php]: :)

Comment (SilverFire) [inline framework/helpers/BaseHtml.php]: :dog:

File: framework/helpers/BaseHtml.php (hunk 3 above)
Comment (githubjeka) [inline framework/helpers/BaseHtml.php]: :+1:

File: framework/helpers/BaseHtml.php (hunk 2 above)
Comment (githubjeka) [inline framework/helpers/BaseHtml.php]: :+1:

File: framework/helpers/BaseHtml.php [hunk 4]
```diff
@@ -974,27 +975,34 @@ public static function radioList($name, $selection = null, $items = [], $options
...
 
-        $separator = isset($options['separator']) ? $options['separator'] : "\n";
+        if (isset($options['separator'])) {
+            $separator = $options['separator'];
+            unset($options['separator']);
+        } else {
+            $separator = "\n";
+        }
+
+        $contentTag = implode($separator, $lines);
```
Comment (TianJinRong) [inline framework/helpers/BaseHtml.php:985]: visibleTag is readble :)

File: framework/helpers/BaseHtml.php [hunk 5]
```diff
@@ -974,27 +975,34 @@ public static function radioList($name, $selection = null, $items = [], $options
...
-        if ($tag === false) {
-            return $hidden . implode($separator, $lines);
-        } else {
-            return $hidden . static::tag($tag, implode($separator, $lines), $options);
+        if (false === $options['tag']) {
+            return $hidden . $contentTag;
         }
+
+        $tag = $options['tag'];
+        unset($options['tag']);
```
Comment (TianJinRong) [inline framework/helpers/BaseHtml.php:1004]: Unset just after use, great!

File: framework/helpers/BaseHtml.php [hunk 6]
```diff
@@ -974,27 +975,34 @@ public static function radioList($name, $selection = null, $items = [], $options
...
+        return $hidden . static::tag($tag, $contentTag, $options);
```
Comment (TianJinRong) [inline framework/helpers/BaseHtml.php:1005]: Returned too much.

**Pull request merged**